src/component-library/Popover/Popover.style.tsx

```diff
@@ -15,6 +15,7 @@ const StyledPopover = styled.div<StyledPopoverProps>`
 
   min-width: 32px;
   min-height: 32px;
+  max-width: calc(100% - 32px);
```

Comment:
Is this an arbitrary value? Wondering why we use a hardcoded value here, but `${theme.spacing.spacing12}` for the modal.

Reply:
You are right. Just fixed it.